.github/workflows/slow-checks.yml

```diff
@@ -38,6 +38,9 @@ jobs:
       - name: "Install all extras"
         run: poetry install --all-extras
 
+      - name: "Setup custom host"
```

Comment:
Uh, yeah you probably need this now

Reply:
That gets fancy when we want to press into pytest fixtures, but for the moment not an issue